docs/resources/aws_ebs_volumes.erb

```diff
@@ -0,0 +1,89 @@
+---
+title: About the aws_ebs_volumes Resource
+platform: aws
+---
+
+# aws\_ebs\_volumes
+
+Use the `aws_ebs_volumes` InSpec audit resource to test properties of some or all AWS EBS volumes. To audit a single EBS volume, use `aws_ebs_volume` (singular).
+
+EBS volumes are persistent block storage volumes for use with Amazon EC2 instances in the AWS Cloud.
+
+Each EBS volume is uniquely identified by its ID.
+
+<br>
+
+## Availability
+
+### Installation
+
+This resource is distributed along with InSpec itself. You can use it automatically.
+
+
+## Syntax
+
+An `aws_ebs_volumes` resource block collects a group of EBS volumes and then tests that group.
+
+    # Ensure you have exactly 3 volumes
+    describe aws_ebs_volumes do
+      its('volume_ids.count') { should cmp 3 }
+    end
+
+    # Use the InSpec resource to enumerate IDs, then test in-depth using `aws_ebs_volume`.
+    aws_ebs_volumes.volume_ids.each do |volume_id|
+      describe aws_ebs_volume(volume_id) do
+        it { should exist }
+        it { should be_encrypted }
+        its('size') { should = 8}
+        its('iops') {should cmp 100}
```

Comment:
This needs spaces after/before the `{}` to satisfy RuboCop.

Reply:
Fixed both lines (plus a couple others lower in that page)